### PR TITLE
Houskeeping

### PR DESCRIPTION
**Pull request by sstok**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -6,5 +6,9 @@ core.autocrlf=lf
 .gitignore export-ignore
 .gitattributes export-ignore
 .travis.yml export-ignore
+.gush.yml export-ignore
```
Comment (cordoval) [inline .gitattributes]: :+1

File: .php_cs [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
-    ->name('*.php')
-    ->ignoreDotFiles(true)
-    ->ignoreVCS(true)
-    ->exclude(array('bin', 'docs'))
-    ->in(__DIR__ . '/src')
-;
-
 return Symfony\CS\Config\Config::create()
-    ->finder($finder)
+    ->setUsingLinter(false)
```
Comment (cordoval) [inline .php_cs]: why not?

Comment (sstok) [inline .php_cs]: It really slows down the process, and the tests already cover the files.

**Comment by cordoval** [comment]
:+1:

**Comment by sstok** [comment]
Merged.

**Pull request merged**